pkg/awx/connection.go

```diff
@@ -137,11 +147,27 @@ func (b *ConnectionBuilder) Build() (c *Connection, err error) {
 		return
 	}
 
+	if b.cacrt != nil && len(b.cacrt) > 0 && b.insecure {
```

Comment:
Here you can use just `len(b.cacrt)` because `len(nil) == 0` (I learned that from @moolitayer).